### ISSUE TITLE
Not ready for epoch 1.0.0?

### ISSUE THREAD

**Issue by ryuyangu** [aeternity/aepp-sdk-python#85]
Since ae has released v1.0.0, I just wonder will this sdk support it?  I tried `./aecli chain top` and got an error  `Unable to understand node reply, perhaps is not an epoch node or is too old?`

**Comment by kryztoval** [comment]
I am using 
```
commit 4cd2b908912cb75a87b8cc3d020d45283b2c700d (HEAD -> develop, origin/develop, origin/HEAD)
Merge: e320c4e c17ee71
Author: Andrea Giacobino <no.andrea@gmail.com>
Date:   Wed Nov 28 19:24:50 2018 +0100

    Merge branch 'develop' of github.com:aeternity/aepp-sdk-python into develop

```

and it seems fine.
```
/opt/aepp-sdk-python # ./aecli chain top
<none>
  Beneficiary _______________________________________ ak_21rna3xrD7p32U3vpXPSmanjsnSGnh6BWFPC9Pe7pYxeAW8PpS
  Hash ______________________________________________ kh_cUX5oQCZ4gUvS9i5bLBbHgQuqpjoHZwAVX2S6XdfuVBTJRrB8
  Height ____________________________________________ 950
  Miner _____________________________________________ ak_2VNqJ98PPmT61Xnc5sjVstBYficNrUzsgEUGA5paqjsNdRaa7k
  Nonce _____________________________________________ 1391888653121407562
  Prev hash _________________________________________ kh_dWg8xgiycZfdbd9EkoC9gg2rsT3BeypK1HU2R62YBLFbPZkRQ
  Prev key hash _____________________________________ kh_dWg8xgiycZfdbd9EkoC9gg2rsT3BeypK1HU2R62YBLFbPZkRQ
  State hash ________________________________________ bs_2wV1wd7m9DSimKM2JDBCdPGx9MrYEaQmYxv2e6JEoagW89dLtW
  Target ____________________________________________ 520794196
  Time ______________________________________________ 2018-11-30T00:53:35.458000+00:00
  Version ___________________________________________ 1
</none>

```

**Comment by kryztoval** [comment]
Oh, there seems to have been a problem with the main server address fixed by #90 
But if you are using the enviromental variables or the -u parameter you should be fine.

Can you try it out?

**Comment by ryuyangu** [comment]
> Oh, there seems to have been a problem with the main server address fixed by #90
> But if you are using the enviromental variables or the -u parameter you should be fine.
> 
> Can you try it out?

Yeah, it is fixed on #90 and  works fine now. Thanks a lot. l'll close this issue.
